### PR TITLE
[documentation] #3945: Update MAINTAINERS.md

### PR DESCRIPTION
**Pull request by 6r1d**
Before merging, please comment on Aleksandr's role, because he's more of an advisor at this point.

### CODE REVIEW

File: MAINTAINERS.md [hunk 1]
```diff
@@ -12,11 +12,11 @@ This is the list of maintainers, including their email address for direct commun
 | Ryu Okada             | [@ryuo88](https://github.com/ryuo88)                 | okada@soramitsu.co.jp           | Product vision                                                                |
```
Comment (Mingela) [inline MAINTAINERS.md:12]:
```suggestion
```

File: MAINTAINERS.md [hunk 2]
```diff
@@ -12,11 +12,11 @@ This is the list of maintainers, including their email address for direct commun
...
 | Ilia Churin           | [@ilchu](https://github.com/ilchu)                   | churin@soramitsu.co.jp          | Development: Rust                                                             |
-| Marin Veršić          | [@mversic](https://github.com/mversic)               | versic@soramitsu.co.jp          | Development: Rust                                                             |
+| Marin Veršić          | [@mversic](https://github.com/mversic)               | versic@soramitsu.co.jp          | Tech lead, development: Rust                                                  |
 | Sam H. Smith          | [@SamHSmith](https://github.com/SamHSmith)           | smith@soramitsu.co.jp           | Development: Rust, C                                                          |
 | Shanin Roman          | [@Erigara](https://github.com/Erigara)               | shanin@soramitsu.co.jp          | Development: Rust                                                             |
 | Shunkichi Sato        | [@s8sato](https://github.com/s8sato)                 | s.sato@soramitsu.co.jp          | Development: Rust                                                             |
```
Comment (Mingela) [inline MAINTAINERS.md:22]:
```suggestion
```

File: MAINTAINERS.md [hunk 3]
```diff
@@ -29,4 +29,4 @@ This is the list of maintainers, including their email address for direct commun
 | Vasily Zyabkin        | [@BAStos525](https://github.com/BAStos525)           | zyabkin@soramitsu.co.jp         | DevOps                                                                        |
 | Ekaterina Mekhnetsova | [@outoftardis](https://github.com/outoftardis)       | mekhnetsova@soramitsu.co.jp     | Documentation                                                                 |
```
Comment (Mingela) [inline MAINTAINERS.md:30]: add Bogdan Yamkovoy?

File: MAINTAINERS.md [hunk 4]
```diff
@@ -12,11 +12,11 @@ This is the list of maintainers, including their email address for direct commun
 | Ryu Okada             | [@ryuo88](https://github.com/ryuo88)                 | okada@soramitsu.co.jp           | Product vision                                                                |
 | Dmitri Venger         | [@dmitrivenger](https://github.com/dmitrivenger)     | venger@soramitsu.co.jp          | Project manager                                                               |
 | Bogdan Mingela        | [@Mingela](https://github.com/Mingela)               | mingela@soramitsu.co.jp         | Iroha Team Lead                                                               |
-| Aleksandr Petrosyan   | [@appetrosyan](https://github.com/appetrosyan)       | petrosyan@soramitsu.co.jp       | Tech lead, development: Rust, C, C++                                          |
+| Aleksandr Petrosyan   | [@appetrosyan](https://github.com/appetrosyan)       | petrosyan@soramitsu.co.jp       | Iroha 2 maintainer, previous tech lead, development: Rust, C, C++             |
```
Comment (Mingela) [inline MAINTAINERS.md:15]:
```suggestion
| Aleksandr Petrosyan   | [@appetrosyan](https://github.com/appetrosyan)       | petrosyan@soramitsu.co.jp       | Development: Rust, C, C++             |
```
Just keep it simple

File: MAINTAINERS.md [hunk 5]
```diff
@@ -12,11 +12,11 @@ This is the list of maintainers, including their email address for direct commun
...
 | Artemii Gerasimovich  | [@QuentinI](https://github.com/QuentinI)             | gerasimovich@soramitsu.co.jp    | Development: Rust                                                             |
 | Daniil Polyakov       | [@Arjentix](https://github.com/Arjentix)             | daniil.polyakov@soramitsu.co.jp | Development: Rust                                                             |
 | Ilia Churin           | [@ilchu](https://github.com/ilchu)                   | churin@soramitsu.co.jp          | Development: Rust                                                             |
-| Marin Veršić          | [@mversic](https://github.com/mversic)               | versic@soramitsu.co.jp          | Development: Rust                                                             |
+| Marin Veršić          | [@mversic](https://github.com/mversic)               | versic@soramitsu.co.jp          | Tech lead, development: Rust                                                  |
```
Comment (Mingela) [inline MAINTAINERS.md:19]:
```suggestion
| Marin Veršić          | [@mversic](https://github.com/mversic)               | versic@soramitsu.co.jp          | Tech lead, development: Rust, Java                                                  |
```

**Comment by Mingela** [comment]
shouldn't we add QAs?

**Comment by 6r1d** [comment]
Yes, I think we should

File: MAINTAINERS.md (hunk 3 above)
Comment (6r1d) [reply MAINTAINERS.md:28]: Thanks, addressed

File: MAINTAINERS.md (hunk 4 above)
Comment (6r1d) [reply MAINTAINERS.md:14]: Given the two uses, be it either finding people who contribute to the project or for Hyperledger administrative processes, I find my current idea to be valid.

Comment (Mingela) [reply MAINTAINERS.md:14]: Commit history always can be an evidence for 'who contributes', please elaborate on the second use

Comment (6r1d) [reply MAINTAINERS.md:14]: TOC elections are a good example here, even if Aleksandr is not a TOC member this time. I think leaving such information is helpful, but if you disagree, I'll remove it.

Comment (appetrosyan) [reply MAINTAINERS.md:14]: > Just keep it simple

Agreed. Architect sounds more appropriate for what I did and continue to do.

Comment (appetrosyan) [reply MAINTAINERS.md:14]: Besides, to be quite honest, I don't really need this document to showcase what I did. Anyone with half a brain will check the commit history, and it's more consistent with an architect than even a technical lead.

**Comment by mversic** [comment]
@Mingela can we merge this as it is?

**Review by mversic — DISMISSED** [review]

**Comment by Mingela** [comment]
@mversic @6r1d should we also add the newcomers?

**Review by Mingela — DISMISSED** [review]

**Comment by mversic** [comment]
> @mversic @6r1d should we also add the newcomers?

yes we just agreed to add Vladislav, is there anyone else?

**Comment by Mingela** [comment]
> > @mversic @6r1d should we also add the newcomers?
> 
> yes we just agreed to add Vladislav, is there anyone else?

the intern maybe, up to you

**Review by mversic — APPROVED** [review]

**Review by Mingela — APPROVED** [review]

**Pull request merged**